bucket/forkgram.json

```
@@ -0,0 +1,30 @@
+{
+    "version": "4.0.0",
+    "description": "Fork of Telegram Desktop messaging app.",
+    "homepage": "http://t.me/forkgram",
+    "license": {
+        "identifier": "GPL-3.0-with-OpenSSL-exception",
```

```suggestion
        "identifier": "GPL-3.0-openssl-exception",
```
https://github.com/spdx/license-list-XML/search?q=openssl+exception